contrib/terraform/gcp/main.tf

```diff
@@ -23,8 +23,10 @@ module "kubernetes" {
 
   master_sa_email  = var.master_sa_email
   master_sa_scopes = var.master_sa_scopes
+  master_preemptible = var.master_preemptible
```

Comment:
nit: I guess we need to align the indent of `=` in the same block like the other blocks.

Reply:
Done.